src/Components/App/App.tsx

```diff
@@ -16,40 +12,42 @@ interface Guesses {
 }
 
 const App: React.FC = () => {
-  const [data, setData] = useState<countryData>()
+  const [data, setData] = useState<string[]>([])
```

Comment:
Can you think of a more descriptive variable than "data"? What does this data represent?